ambari-server/src/main/python/ambari_server/serverSetup.py

```diff
@@ -1272,6 +1272,17 @@ def setup_jce_policy(args):
   print 'NOTE: Restart Ambari Server to apply changes' + \
         ' ("ambari-server restart|stop|start")'
 
+def get_java_major_version(cmd_out):
+  version_short = re.split("[java|openjdk|.*] version", cmd_out)[1].split(" ")[1][1:-1]
+  if re.match("1\.8.*", version_short): # 1.8.0_112
```

Comment:
While Ambari does not support Java 1.7 and earlier, the function should work for those, too.  Otherwise setup will fail with exception (and so does unit test).

```
Fatal exception: Running java version check command failed: list index out of range. Exiting., exit code 1'
```